packages/packages/documents/src/types.ts

```diff
@@ -1,19 +1,19 @@
 import { createSlice } from './store';
 
 export type State = {
-	documents: Record<Document['id'], Document>,
-	currentDocumentId: Document['id'],
+	entities: Record<Document['id'], Document>,
+	activeId: Document['id'],
 }
 
 export type Document = {
 	id: number,
 	title: string,
-	status: string, // TODO: union?
-	isModified: boolean,
+	status: 'publish' | 'future' | 'draft' | 'pending' | 'private' | 'trash' | 'auto-draft' | 'inherit',
```

Comment:
Extract into its own type and export it for others to use

Reply:
Done